org.osgi.test.common/src/main/java/org/osgi/test/common/annotation/ServicePropertyConverter.java

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright (c) OSGi Alliance (2020). All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.osgi.test.common.annotation;
+
+import java.lang.reflect.Array;
+import java.util.Arrays;
+import java.util.Dictionary;
+import java.util.Hashtable;
+
+import org.osgi.test.common.annotation.ServiceProperty.Scalar;
+import org.osgi.test.common.annotation.ServiceProperty.Type;
+
+public class ServicePropertyConverter {
```

Comment:
This would then be named `PropertiesConverter`.

Reply:
ok